api/src/main/java/datawave/microservice/querymetric/BaseQueryMetric.java

```diff
@@ -682,6 +683,10 @@ public int getFieldNumber(String name) {
     @XmlElement(name = "prediction")
     protected Set<Prediction> predictions = new HashSet<>();
     
+    @XmlElement(name = "subplans")
+    @XmlJavaTypeAdapter(StringMapAdapter.class)
+    protected Map<String,String> subPlans = new HashMap<>();
```

Comment:
Now that I have seen this all the way through the html/json/xml representations, I think we are going to have a problem with the volume of ranges.  I would like to refine this to be a Map<String, int[2]> or a Map<String, RangeCounts> where range counts contains one counter for document ranges and another for shard ranges.  This will SIGNIFICANTLY reduce the amount of data in the query metric and will allow this to remain performance.